.github/workflows/release_notes.yml

```
@@ -0,0 +1,42 @@
+
+permissions:
+  contents: read
+
+jobs:
+  regen-notes:
+    name: 'Regenerate release notes'
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Update notes
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:135: Double quote to prevent globbing and word splitting

(shellcheck)